src/ng-select/lib/ng-select.component.ts

```diff
@@ -889,13 +897,14 @@ export class NgSelectComponent implements OnDestroy, OnChanges, OnInit, AfterVie
 	private _updateNgModel() {
 		const model = [];
 		for (const item of this.selectedItems) {
-			if (this.bindValue) {
+			const bindValue = this.bindValue();
```

Comment:
Nit: I don't we need separate variable here

Reply:
Updated - https://github.com/ng-select/ng-select/pull/2584/commits/53cf59e5182d0cbc93ac9608a82ee5b0dc0c44be